server/src/main/java/org/cloudfoundry/identity/uaa/oauth/UaaTokenStore.java

```diff
@@ -214,7 +214,7 @@ protected void performExpirationClean() {
         Instant last = lastClean.get();
```

Comment:
This lastClean instance - before it was only a value needs to be set after first usage, 
To me this caused the issue, that the check in line 217 always was true and cleanup is executed always.
Even with lastClean.compareAndSet(.....) the lastClean is not set to a recent Instant but still in year 1970....